File: tests/dart/sample.dart
const int max = 100;
final String name = "app";

class Greeter {
  final String name;
  Greeter(this.name);

  String greet() => "hi $name";
  Future<void> load() async {}
}

abstract class Shape {
  double area();
}

int add(int a, int b) => a + b;

File: tests/kotlin/sample.kt
const val MAX = 5

class Greeter(val name: String) {
    fun greet(): String = "hi $name"
    suspend fun load() {}
}

data class Point(val x: Int, val y: Int)

interface Shape {
    fun area(): Double
}

object Registry {
    fun register() {}
}
